hive/ranger-client.sh

```diff
@@ -49,26 +67,46 @@ function install_ranger() {
 
         URL=http://${HOST}:${PORT}
         HIVE_LIB=/usr/lib/hive1.2
+
+        S3_PATH=s3://paid-qubole/ranger-${PLUGIN_VER}
+        RANGER_PLUGIN=ranger-${PLUGIN_VER}-hive-plugin
+        RANGER_PLUGIN_FILE=${S3_PATH}/${RANGER_PLUGIN}.tar.gz
+        SPOOL_DIR=/media/ephemeral0/logs/ranger/hive/audit/solr
```

Comment:
nit: Move this to [here](https://github.com/qubole/bootstrap-functions/pull/35/files#diff-3b67c313e5a3b52155adacfce2d1980eR99) as it is only used in `if` block